src/app/theme/components/baMenu/baMenu.service.ts

```diff
@@ -1,5 +1,24 @@
 import {Injectable} from '@angular/core';
 import {Router, Routes} from '@angular/router';
+export interface INgAdminMenuItem {
```

Comment:
@mterwoord could you remove the I prefix as per angular style guide? https://angular.io/styleguide#!#03-03